src/components/table/table-header/table-header.component.js

```diff
@@ -155,15 +155,22 @@ class TableHeader extends React.Component {
     } else {
       contents = this.props.children;
     }
+
+    const headerProps = {
+      align: this.props.align,
+      sortable: this.props.sortable
+    };
+
+    if (/^\d+$/.test(this.props.width)) headerProps.width = this.props.width;
```

Comment:
Would `!isNaN(this.props.width)` work? If so that might read better

Reply:
Yeah good point, the linter was complaining when I tried using `isNaN`, but just realised you can use `Number.isNaN`

Reply:
👌